README.md

```diff
@@ -13,37 +30,47 @@ npm i @cloudflare/kv-asset-handler
 
 This package was designed to work with [Worker Sites](https://workers.cloudflare.com/sites).
 
-### `getAssetFromKV`
+## `getAssetFromKV`
 
-`getAssetFromKV` is a function that takes a `FetchEvent` object and returns a `Response` object if the request matches an asset in KV, otherwise it will throw an `Error`.
+getAssetFromKV(FetchEvent) => Promise<Response>
+
+`getAssetFromKV` is an async function that takes a `FetchEvent` object and returns a `Response` object if the request matches an asset in KV, otherwise it will throw a `KVError`.
 
 
 #### Example
 
 This example checks for the existence of a value in KV, and returns it if it's there, and returns a 404 if it is not. It also serves index.html from `/`.
 
+### Return
+
+`getAssetFromKV` returns a `Promise<Response>` with `Response` being the body of the asset requested.
```

Comment:
```suggestion
`getAssetFromKV` returns a `Promise<Response>` where the `Response` body contains the asset requested. It will reject with one of the KVError variants listed below.
```